app/Services/UserService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Services;
+
+class UserService extends Service {
+    
+    public function createUser($data = array()) {
+        $db = $this->Db;
+        
+        $username = $data['username'];
+        $email = $data['email'];
+        
+        $query = $db->prepare("INSERT INTO users VALUES (NULL, ?, ?, NOW())");
+        $query->bindParam(1, $username, \PDO::PARAM_STR);
+        $query->bindParam(2, $email, \PDO::PARAM_STR);
+        if(!$query->execute()) {
+            throw new Exception("Something wrong with your query");
+        }
+        
+        return true;
+    }
+    
+    public function getUser($id) {
+        $db = $this->Db;
```

Comment:
please remove $db var initiatialization and use $db property from base sevice instead, for instance: $this->db->query(...